build-tools/validate-e2e

```diff
@@ -18,7 +18,7 @@ endpoint=http://localhost:$PORT
 # We need to wait a bit for the server to come up. curl for a max time
 echo "Waiting for server to start..." >&2
 up=false
-for i in $(seq 0 100); do
+for i in $(seq 0 1000); do
```

Comment:
Good question! The problem with a fixed `sleep` is that for some it might take too long, for others it might still be too short. Inelegant as it may appear at first sight, retrying n times seems to be the most time-efficient way (from the perspective of the user running the tests) of checking whether the server is up already.

I recently did some work related to this in another project and using this approach of retrying (rather than having a fixed time wait) did wonders to reduce waits.

So I suggest we leave it like this, only with more retries for slower computers like mine :D